README.md

```diff
@@ -1,2 +1,33 @@
 # leto-modelizer-templates-library
-Library that contains all default template for leto-modelizer.
+
+Library that contains all default templates for leto-modelizer.
+
+## Create a template
+
+1. Create the metadata of your templates in the [index.json](index.json):
+
+metadata structure:
+```json
+{
+    "key": "terraform_webapp",
+    "name": "Web application",
+    "plugin": "terrator-plugin",
+    "type": "model",
+    "description": "Schema of web application.",
+    "url": null,
+    "files": ["main.tf"]
+}
+```
+
+- `key` attribute is the folder name of your template under the `templates` folder.
+- `name` attribute is the display name used in leto-modelizer.
+- `plugin` attribute is the plugin name used by the template.
+- `type` attribute is the type template, either `model` or `component`.
+- `description` attribute is the description used in leto-modelizer to explain the usage of this template.
+- `url` attribute is an url attached to a documentation of this template used in leto-modelizer.
+- `files` attribute is the list of files used in the template.
+
+2. Create the folder of your template with the choosen key you provide in the [index.json](index.json)
```

Comment:
choosen -> chosen key you provide -> provided

Reply:
done